src/utils/CustomBox.js

```diff
@@ -1,33 +1,43 @@
 const chalk = require('chalk');
```

Comment:
This probably should be moved entirely in util helpers or in another class of Utils so able to be exported.